src/components/common/SecureRouteAuth0.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Nice job making this component! It is nice and modular, making it easy to implement.